static/src/stylesheets/inline/story-package-garnett.scss

```diff
@@ -28,7 +34,37 @@
 }
 
 .fc-container--story-package {
-    background-color: #6b6e71; // This colour does not exist (in the palette)
+    background-color: $dynamo-background;
+
+    .fc-container__body {
+        &:before {
+            @include mq($from: leftCol) {
+                position: absolute;
+                content: '';
+                display: block;
+                top: 0;
+                bottom: -12px;
+                width: 1px;
+                background-color: #929698;
+                left: 170px; //magic numbers sorry --\_()_/--
+            }
+
+            @include mq($from: wide) {
+                left: 250px; //magic numbers sorry --\_()_/--
+            }
+        }
+    }
+
+    &.js-container--first {
```

Comment:
We should try not to hang css styles off of `js-` classes, you can use `fc-container--first`

Reply:
Done